### ISSUE TITLE
BeefLSP Bug

### ISSUE THREAD

**Issue by Mezo-hx** [beefytech/Beef#1896]
The bug basically throws this error when attempting to build the BeefLSP

```
Execution Failed
Executing Command: CopyToDependents("$(ProjectDir)/dist/libBeefySysLib_d.so")
ERROR: BUILD FAILED
```

Which was caused by this line
<https://github.com/MineGame159/Beef/blob/lsp/BeefLsp/src/Program.bf#L27>

**Comment by hermansimensen** [comment]
Still an issue with the latest Beef. It seems there is a compiler issue related to the specified line. Specifically it seems like invoking the const string is what is causing the compiler to fail.

**Comment by bfiete** [comment]
Fixed at 0bed50cf7c5ca441f3ad815c38f0e42fe462634a
